tests/client/api_events/test_milestones_confirmed_event.c

```diff
@@ -72,6 +75,9 @@ void test_milestones_confirmed_events(void) {
   TEST_ASSERT_NOT_NULL(client);
   TEST_ASSERT_EQUAL_INT(0, event_register_cb(client, &callback));
   TEST_ASSERT_EQUAL_INT(0, event_start(client));
+  // Wait until test is completed
+  while (!test_completed)
```

Comment:
if no event was received by the MQTT client, this test case will be an infinite loop.
maybe add a timeout?

Reply:
Done.